### PR TITLE
Bump to Buildah v1.25.1, openshift/imagebuilder v1.2.3

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
[NO NEW TESTS NEEDED]

This bumps Buildah to v1.25.1 and openshift/imagebuilder to v1.2.3

Signed-off-by: tomsweeneyredhat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
I'm leery about adding the "replace" command for containers/common, but for the moment, it's the quickest way around a nasty vendor dance that we're undergoing.  We should talk at standup tomorrow about the state of the c/common main branch and plans for the near future for it and the other branches.  It's not clear to me at least.

@Luap99 @vrothberg PTAL

**Comment by edsantiago** [comment]
This is badly, badly broken. Common factor, best I can tell, is:
```
# podman build SOMETHING
STEP 1 FROM ... <----- is ok
STEP 2 RUN some command     <---- this is what fails
error running container: setting file label on "/proc/self/fd/12": lsetxattr /proc/self/fd/12: operation not supported
```

**Comment by TomSweeneyRedHat** [comment]
The problem seems to be pulling in this commit from c/common https://github.com/containers/common/pull/971 and then the changes in Podman to use that.  I tried to back that out of Podman temporarily, but I'm too tired to see straight at this point.  @Luap99 or @vrothberg if you could take a look when you get in, I'd appreciate it.

File: go.mod [hunk 1]
```diff
@@ -45,16 +45,15 @@ require (
...
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.18.1
+	github.com/onsi/gomega v1.19.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.3-0.20220114050600-8b9d41f48198
 	github.com/opencontainers/runc v1.1.0
 	github.com/opencontainers/runtime-spec v1.0.3-0.20211214071223-8958f93039ab
 	github.com/opencontainers/runtime-tools v0.9.1-0.20220110225228-7e2d60f1e41f
-	github.com/opencontainers/selinux v1.10.0
+	github.com/opencontainers/selinux v1.10.1
```
Comment (Luap99) [inline go.mod:54]: You have to remove this change to fix the tests, see https://github.com/containers/podman/pull/13687 https://github.com/containers/buildah/pull/3875

**Comment by flouthoc** [comment]
@TomSweeneyRedHat I think regression is in `selinux v1.10.1` which was bumped here https://github.com/containers/common/pull/987 and I think maybe because of this https://github.com/opencontainers/selinux/pull/173, reverting `selinux v1.10.0` should fix this issue: https://github.com/containers/common/pull/994

You can try by pointing `c/common` to my fork just to verify
```console
go mod edit -replace github.com/containers/common=github.com/flouthoc/common@master
```
and revert selinux in go.mod to `v1.10.0`

File: go.mod (hunk 1 above)
Comment (flouthoc) [reply go.mod:54]: @Luap99 I think we might wanna revert version bump of selinux in `c/common` cause `make vendor` will automatically bump `selinux`. Could you review my above PR.

**Comment by flouthoc** [comment]
@TomSweeneyRedHat I think you can do `go mod edit -replace github.com/opencontainers/selinux=github.com/opencontainers/selinux@v1.10.0` till this is fixed or reverted in `c/common`

**Comment by TomSweeneyRedHat** [comment]
Closing, replaced by #13733

**Pull request closed without merging**